### DISCUSSION TITLE
About Database

### DISCUSSION THREAD

**Discussion by mdxabu** [KathiraveluLab/Beehive#2] [Q&A]
Dear @pradeeban,

I hope this message finds you well. I was wondering if we are using a cloud MySQL database or if we are utilizing a different system?

Looking forward to your response.

Best regards,
F Mohamed Abdullah

**Comment by pradeeban** [comment]
Hi @mdxabu,
Thanks for your message,
Both MySQL and Mongo (NoSQL datastores) have their benefits. Given this is a new project, we have a high degree of freedom to choose the technology and frameworks at this point.

Regards,
Pradeeban.
